src/services/comments.js

```diff
@@ -53,22 +53,22 @@ module.exports = class CommentsHelper {
 				return responses.successResponse({
 					statusCode: httpStatusCode.ok,
 					message: 'COMMENT_UPDATED_SUCCESSFULLY',
-					result: commentCreate,
+					result: {},
```

Comment:
@adithyadinesh0412 since there can be multiple comments, we don't need to update the status in the reviews table.